doc/swish/erlang.tex

```diff
@@ -1394,6 +1405,27 @@ \subsection{Miscellaneous}
 for the file \var{path} using \var{mode}, which defaults to
 \code{\#o777}. It returns \var{path}.
 
+% ----------------------------------------------------------------------------
```

Comment:
Good ideas! I updated the osi-port functions in io.ss to raise an osi-port-closed tuple instead of the unhelpful invalid foreign-procedure argument error.